### ISSUE TITLE
Review Final Projects

### ISSUE THREAD

**Issue by syahrulhamdani** [sakinahindriyani/H8_PTP_Final_Image#1]
Well done! You've done well especially in the documentation. Some improvements you may want to consider:
1. It's been the convention to define all train, valid, and test set in the early cells, instead of defining test set in the later cells. You may want to move `test = data_generator...` up.
2. If you want to use your model more than once, it's good to wrap it in a function. Remember the DRY, Don't Repeat Yourself. Also, do the same when training the model.
3. You may want to give a disclaimer which TensorFlow version you use.

Overall you've done great!
